src/community/resources/index.md.njk

```diff
@@ -80,6 +80,12 @@ A Rails form builder using GOV.UK Frontend
 
 ## Write code
 
+[GOV.UK Design System snippets Atom package](https://atom.io/packages/govuk-design-system-snippets)⁠ - 
```

Comment:
Could you add 2 spaces after the final hyphen? (it create a line break in the HTML)